### ISSUE TITLE
windows compatibility sbt 0.7.x issues

### ISSUE THREAD

**Issue by jberkel** [jberkel/android-plugin#5]
http://groups.google.com/group/simple-build-tool/browse_thread/thread/29fe4de0309cc389

```
    [info] == AndroidBaby / aapt-generate == 
    java.lang.RuntimeException: Could not parse command: (1,3): Escape sequences 
    can only occur in a quoted argument 
    at scala.Predef$.error(Predef.scala:76) 
    at sbt.Process$.apply(Process.scala:26) 
    at sbt.Process$.apply(Process.scala:15) 
    at sbt.Process$.apply(Process.scala:40) 
    at AndroidProject$$anonfun$aaptGenerateTask$1.apply(AndroidProject.scala:97) 
    at AndroidProject$$anonfun$aaptGenerateTask$1.apply(AndroidProject.scala:97) 
    at sbt.ExecProject$$anonfun$execTask$1.apply(ScalaProject.scala:425) 
    at sbt.ExecProject$$anonfun$execTask$1.apply(ScalaProject.scala:424) 
    at sbt.TaskManager$Task.invoke(TaskManager.scala:62) 
    at sbt.impl.RunTask.doRun$1(RunTask.scala:77) 
    at sbt.impl.RunTask.runTask(RunTask.scala:85) 
    at sbt.impl.RunTask.sbt$impl$RunTask$$runIfNotRoot(RunTask.scala:60) 
    at sbt.impl.RunTask$$anonfun$runTasksExceptRoot$2.apply(RunTask.scala:48) 
    at sbt.impl.RunTask$$anonfun$runTasksExceptRoot$2.apply(RunTask.scala:48) 
    at sbt.Distributor$Run$Worker$$anonfun$2.apply(ParallelRunner.scala:131) 
    at sbt.Distributor$Run$Worker$$anonfun$2.apply(ParallelRunner.scala:131) 
    at sbt.Control$.trapUnit(Control.scala:19) 
    at sbt.Distributor$Run$Worker.run(ParallelRunner.scala:131) 
    [info] == AndroidBaby / aapt-generate ==
```

**Comment by jberkel** [comment]
should be fixed with sbt 0.7.2. don't have windows installed, could anybody please confirm so I can close the bug?

**Comment by jberkel** [comment]
no other reports, closing
